chart/infra-server/static/flavors.yaml

```diff
@@ -139,7 +139,8 @@
       value: ""
       kind: optional
       help: |
-        e.g. 1.19.12-gke.2100. Use 'gcloud container get-server-config --zone=us-central1 --project acs-team-temp-dev' to see all versions.
+        e.g. 1.19.12-gke.2100. Use `gcloud container get-server-config
+        --zone=us-central1 --project acs-team-temp-dev` to see all versions.
```

Comment:
1. Because select-and-copy can include adjacent punctuation, like final period in bot comment:

    ```
    To use with deploy scripts, first export MAIN_IMAGE_TAG=4.3.x-155-gd0f4f25356.
    ```

2. And applying a pattern from instructional and technical writing to write **why** before **how**

What do you think about 3 lines? Best if output preserves line break, but improvement even if not.

```
e.g. 1.19.12-gke.2100
To see all versions:
gcloud container get-server-config --zone=us-central1 --project acs-team-temp-dev
```